src/CreateDailyWeather.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const daysOfWeek = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+
+export default function CreateDailyWeather({ info }) {
+  const weatherIconURL = `https://openweathermap.org/img/wn/${info.iconKey}@2x.png`;
+
+  const today = new Date(info.timestamp * 1000);
+  const currentDay = today.getDay();
+
+  return (
+    <div className="col border border-primary rounded text-center m-1">
+      <h4>{daysOfWeek[currentDay]}</h4>
+      <img src={weatherIconURL} alt="Icon representing the current weather" />
```

Comment:
I think with the alt text as it is, someone with a screen reader wouldn't understand what the weather conditions are, since there also isn't a matching description displayed.  

Since each icon has a corresponding `description`, could you include this somehow, possibly in the alt text?

Reply:
Will do! Definitely have to make this more accessibility.